source/draft_api.rst

```diff
@@ -600,7 +650,17 @@ Response body:
        "created_at":"2014-04-17",
        "updated_at":null,
        "deleted_at":null,
-       "revision":1
+       "revision":1,
+       "users": {
+         "members": [
+           "patcht"
+         ],
+         "spectators": [
+         ],
+         "managers": [
+           "tschuy"
+         ]
+       }
     }
 
 Note that this endpoint, when called, will automatically set the currently
```

Comment:
"this endpoint, when called, will automatically set the currently authenticated user as a member, spectator, and manager of the project"